client/task-list/tasks/payments/PaymentGatewaySuggestions/components/PaymentAction.js

```diff
@@ -0,0 +1,125 @@
+/**
```

Comment:
I went ahead and separated this on the basic of local/remote payments. The old file is currently called `PaymentActionLocal`, which isn't great, but I anticipate it going away pretty quick.